internal/photoprism/mediafile.go

```diff
@@ -332,6 +332,17 @@ func (m *MediaFile) ExtractEmbeddedVideo() (string, error) {
 
 	// Get the embedded video field name from the file metadata.
 	if metaData := m.MetaData(); metaData.Error == nil && metaData.EmbeddedVideo != "" {
+		// Check if an embedded video sidecar already exists
+		if embeddedVideo := m.HasEmbeddedVideo(); embeddedVideo != "" {
+			return embeddedVideo, nil
+		}
+
+		// Check if the camera make is supported
+		if _, ok := entity.CameraMakes[metaData.CameraMake]; !ok {
```

Comment:
`entity.CameraMakes` is just a lookup table for normalizing the vendor string. It might be best to encapsulate the validation in the `meta` package, so that `metaData.EmbeddedVideo` is only set if the original media file can (optionally) contain an MP4 video. If you need a list of vendor strings (or other metadata properties) to check against, it would make sense to add these to the `meta` package as well, e.g. as string constants or a `map[string]bool` lookup table.